central/reprocessor/reprocessor.go

```diff
@@ -433,22 +433,31 @@ func (l *loopImpl) reprocessImagesAndResyncDeployments(fetchOpt imageEnricher.Fe
 func (l *loopImpl) reprocessNode(id string) bool {
 	node, exists, err := l.nodes.GetNode(allAccessCtx, id)
 	if err != nil {
-		log.Errorf("error fetching node %q from the database: %v", id, err)
+		log.Errorf("fetching node id %q from the database: %v", id, err)
 		return false
 	}
 	if !exists {
-		log.Warnf("node %q does not exist in the database. Skipping...", id)
+		log.Warnf("fetching node id %q from the database: node does not exist", id)
+		return false
+	}
+
+	if nodeEnricher.SupportsNodeScanning(node) {
+		log.Infof("node %s:%s (id: %s) does not need reprocessing (non-legacy node)",
```

Comment:
What does non-legacy node mean? If we skip enrichment, how do we make ensure the snoozed state of vulns is updated?